fhir-server/src/main/java/com/ibm/fhir/server/resources/FHIRResource.java

```diff
@@ -4211,24 +4220,24 @@ private URI toUri(String uriString) throws URISyntaxException {
 
     private Response buildResponse(FHIROperationContext operationContext, Resource resource)
         throws URISyntaxException {
-        
+
         // The following code allows the downstream application to change the response code
-        // This enables the 202 accepted to be sent back 
+        // This enables the 202 accepted to be sent back
         Response.Status status = Response.Status.OK;
         Object o = operationContext.getProperty(FHIROperationContext.PROPNAME_STATUS_TYPE);
         if(o != null) {
             status = (Response.Status) o;
-            
-            
+
+
             if (Response.Status.ACCEPTED.compareTo(status)==0) {
```

Comment:
mind switching per the Project pattern to .equals?

Reply:
sounds good!